.circleci/config.yml

```diff
@@ -13,10 +13,10 @@
         - v1-dependencies-
     - run:
         name: install dependencies
-        command: ./gradlew :xio:dependencies
+        command: ./gradlew dependencies
     - run:
         name: run build test verify
-        command: ./gradlew :xio:build
```

Comment:
we want to run the `ReverseProxyFunctionalTest` and others (?) which is/are not in the xio gradle module